localstack/create-tables-dynamodb.sh

```diff
@@ -230,6 +230,5 @@ EOF
 )
 
 echo $(aws --endpoint-url ${ENDPOINT} dynamodb create-table --cli-input-json "$TABLE_SCHEDULE")
-
-
+echo $(aws --endpoint-url ${ENDPOINT} dynamodb update-time-to-live --table-name "$TABLE_SCHEDULE" --time-to-live-specification Enabled=true,AttributeName=e_at --region us-east-1)
```

Comment:
I am not sure why it is needed.

Reply:
to create TTL, because I get errors when trying to create tables